packages/superset-ui-chart-controls/src/shared-controls/index.tsx

```diff
@@ -495,6 +540,11 @@ const sharedControls = {
   adhoc_filters: enableExploreDnd ? dnd_adhoc_filters : adhoc_filters,
   color_scheme,
   label_colors,
+  x_axis_title,
```

Comment:
Could we define a `section` to share all of these, please refer to: https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-chart-controls/src/sections/forecastInterval.tsx

Reply:
Done